screens/user-list.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ *
+ * Generated with the TypeScript template
+ * https://github.com/react-native-community/react-native-template-typescript
+ *
+ * @format
+ */
+import React, { useState } from 'react';
+import { View, Text, FlatList, StyleSheet, ActivityIndicator } from 'react-native';
+import { NetworkStatus, useQuery } from '@apollo/client';
+import { getUsers } from '../components/utils/graphql-requests';
+
+const UserListScreen = () => {
+  const [offset, setOffset] = useState(0);
+
+  const { loading, error, data, refetch, networkStatus } = useQuery(getUsers, {
+    variables: { offset: offset },
+    notifyOnNetworkStatusChange: true,
+  });
+
+  const [users, setUser] = useState(data?.users.nodes);
+
+  const UpdateOffset = () => {
```

Comment:
```suggestion
  const handleEndReached = () => {
```

Temos uma convenção de usarmos, para esse tipo de callback de eventos de tela, uma abordagem declarativa (o trigger que faz essa função rodar). É diferente da abordagem imperativa que costuma ser mais comum, em que nomeamos função pelo que ela faz. Repare no `camelCase` (convenção do javascript) e no prefixo `handle` (também parte, da nossa convenção).